site/docs/src/json/applications/search-post-request.json

```diff
@@ -0,0 +1,10 @@
+{
+  "search": {
+    "name": "registration",
```

Comment:
Just for readability, should we use a value that is more obviously an application name such as `Payroll` or `Support portal`?

Reply:
👍